edx_lint/files/common_constraints.txt

```diff
@@ -10,3 +10,6 @@
 
 # using LTS django version
 Django<2.3
+
+# Pinned social-auth-core to <4.0.3 untill drf-jwt adds support for PyJWT>2.0
```

Comment:
Could you also add a link to https://github.com/Styria-Digital/django-rest-framework-jwt/issues/76 , which I think is the upstream ticket for this?

Reply:
Added